### ISSUE TITLE
Edge firewall port ranges

### ISSUE THREAD

**Issue by axl303** [ovh/infrastructure-roadmap#219]
As the title and as a customer of OVH, I want and need to be able to use port ranges for Edge Firewall rules (dst & source).

By using port ranges into one rule, we may also save some rules.

**Comment by Gawnz1** [comment]
I want to add as a suggestion and multi port (not sure if this is the right to name it)
Like the iptables -m multiport, so one rule for example in the Edge firewall can become: Example: destination port, 80,443 or even more, TCP destination port 22,80,443 and so on..

**Comment by CoffeSiberian** [comment]
This feature would be very useful, it's a shame to see it paused...

**Comment by jslocinski** [comment]
We are working to release port-range support for Edge Network Firewall in comming months. For now multiport option is not possible.

**Comment by MikeRuSe** [comment]
I do agree that this feature would be super useful and most of the resellers will be able to benefit from it. Due to the current limitation of rules, some customers are currently allowing all TCP and all UDP traffic because their amount of consecutive ports to open exceed the number of rules.
In other words, port ranges at Edge Firewall is a must and super needed (without this some customers may end up accidentally exposing services they don’t want to the internet)

**Comment by sudo-nitz** [comment]
> We are working to release port-range support for Edge Network Firewall in comming months. For now multiport option is not possible.

Hi!
What is the current status of this functionality? I agree with other replies that is very needed.

**Comment by Akouniza** [comment]
Hi!
Up about this feature 👍

**Comment by MikeRuSe** [comment]
Hi folks, unless these details are outdated, this is planned to happen EoM (Feb 28th). Just in case, @jslocinski could you please confirm if these details are up to date? If not, is there a more accurate ETA? Thanks!

**Comment by Akouniza** [comment]
Hi thanks for your reply! 
Perfect! I was wondering if it was still in the plan :D

**Comment by mrc4tt** [comment]
It's still planned as Release date hasn't been changed (You can already make changes by API)

/Miksen

**Comment by jslocinski** [comment]
Yes, we're going to release it during next weeks in the UI. Moreover in the APIv6 it's already available ^^

**Comment by brendancarr** [comment]
I am absolutely done with your customer support and their run-arounds regarding the edge firewall, and their inability to escalate a ticket to solve an issue with a server being flooded through Apache. I have a WHM server and it requires more than 20 ports to be opened. Currently the edge firewall has ALL ports opened because I have to, not because I want to. I have had to use mod_evasive and a combination of CSF, Cloudflare, and geo-blocking rules to keep it online. Anxiously awaiting the day it launches and I can stop dealing with this person!

**Comment by Howtek84** [comment]
The date 28.02.2026 has already passed, yet I still cannot use this feature in my user panel. Why ?

**Correction:** I noticed that it has been added to the API side; this will meet my needs.

**Comment by jslocinski** [comment]
Hello, feature is now live in Control Panel:

<img width="1021" height="160" alt="Image" src="https://github.com/user-attachments/assets/8245acf5-b2c1-4167-9e40-ae53fd41bce6" />

**Comment by Ryeera** [comment]
I don't know if the feature is working for me or not because I can't enable the firewall when I don't know if it's working as this is a productive environment but I did notice that while I can enter port-ranges, it's not displaying them after I created the rule. Is this a known issue?

In my case I entered the bottom three rules with port ranges.

<img width="1757" height="489" alt="Image" src="https://github.com/user-attachments/assets/f355b0d9-a7de-4722-975f-cf1bd3d9e8da" />

**Comment by jslocinski** [comment]
Could you please clean-up your browser cache and test again?

**Comment by Ryeera** [comment]
> Could you please clean-up your browser cache and test again?

Have done that and even tried an incognito-tab, none of which worked.

**Comment by jslocinski** [comment]
Weird, you are still using the old interface.
Lets try this way: go to https://manager.eu.ovhcloud.com/#/network/ip , search for your IP, click at the "Edge Firewall" status badge to edit rules.

**Comment by Ryeera** [comment]
> Weird, you are still using the old interface. Lets try this way: go to https://manager.eu.ovhcloud.com/#/network/ip , search for your IP, click at the "Edge Firewall" status badge to edit rules.

Ah that might be it, I'm using the US OVH. The login doesn't work in EU and also looks different. I currently have the "Beta Navigation" enabled on the top, but the "Classic navigation", while looking different, also doesn't show more information.

**Comment by jslocinski** [comment]
US version sometimes may have a little delay, but early next week it will be up & running.

**Comment by Ryeera** [comment]
So the issues I described a month ago still applies. Please advise.
